### PR TITLE
Removed the warning options that prevent compiling.

### PR DESCRIPTION
**Pull request by jamesorr**
Everything should now compile.  It turns out some of the new warning options in derivnum.R and errors.R prevented it from compiling.  I've removed those for now, so some warnings still show up when T or S are out of range (when calling one of those routines).  But overall things are much better, because most the warnings have been supressed.

This is all more complicated than I thought because in those 2 routines Jean-Marie has done some fancy programming to make local copies of some of the routines (e.g., for each of the constants) so he can perturb them in an efficient way (to calculate derivatives) without changing the carb routine.  But to do that, his local routines refer to the original seacarb routines in the "seacarb::" environment.  So I can't refer to a new 'warn' option in those older routines because they don't have it yet; they are not my updated versions.  

In short, everything should compile now and most of the warnings have been supressed. I'll talk to Jean-Marie when he returns from vacation to see what mght be done to supress the remaining warnings. This is not critical for now.

### CODE REVIEW

**Comment by jpgattuso** [comment]
Dear Jim,

All seems well.

I have made some changes and additions on several man files. Could you 
check these as well as the ChangeLog file? I propose that you replace my 
email address by yours as the contact for the new functions. I would be 
embarrassed to reply to technical queries!

When shall we upload the new version on CRAN?

I am attaching the package but, as before, changes should be made on git 
(where the files are up-to-date.

Thanks again for the great additions! The next step will be Hagens’ 
but I want to complete the present version first. Handling two at the 
same time would be too complicated.

Best regards,
Jean-Pierre




On 6 Aug 2016, at 0:30, James Orr wrote:

> Everything should now compile. It turns out some of the new warning 
> options in derivnum.R and errors.R prevented it from compiling. I've 
> removed those for now, so some warnings still show up when T or S are 
> out of range (when calling one of those routines). But overall things 
> are much better, because most the warnings have been supressed.
> This is all more complicated than I thought because in those 2 
> routines Jean-Marie has done some fancy programming to make local 
> copies of some of the routines (e.g., for each of the constants) so he 
> can perturb them in an efficient way (to calculate derivatives) 
> without changing the carb routine. But to do that, his local routines 
> refer to the original seacarb routines in the "seacarb::" environment. 
> So I can't refer to a new 'warn' option in those older routines 
> because they don't have it yet; they are not my updated versions. 
> In short, everything should compile now and most of the warnings have 
> been supressed. I'll talk to Jean-Marie when he returns from vacation 
> to see what mght be done to supress the remaining warnings. This is 
> not critical for now.


---------------------------------------------
Jean-Pierre Gattuso | http://www.obs-vlfr.fr/~gattuso
